openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -201,7 +201,16 @@
     "end-date": "End Date",
     "show-deleted": "Show deleted",
     "add-bot": "Add Bot",
-    "search-for-bots": "Search for bots..."
+    "search-for-bots": "Search for bots...",
+    "condition-is-invalid": "Condition is invalid",
+    "rule-name": "Rule Name",
+    "write-your-description": "Write your description",
+    "select-resource": "Select Resources",
+    "name-is-required": "Name is required",
+    "resources-is-required": "Resources are required",
+    "operations-are-required": "Operations are required",
```

Comment:
For such let's create generic labels like:
```
"field-required": "{{field}} is required",
"field-required-plural": "{{field}} are required"
```

Reply:
Sure I will update it.

Reply:
Then we can use it like this right?
```js
t('label.field-required', { field: 'name' }),
```